lib/xcake/generator/build_phase_registry.rb

```diff
@@ -14,6 +14,7 @@ module Registry
         #
         def self.build_phase_generators
           [
+              XCConfigIgnorePhase,
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left square bracket is.